### PR TITLE
Keep Gatsby around

### PR DESCRIPTION
**Pull request by mxdvl**
# What does this change? 

Keep all 199 commits made towards the never-released Gatsby version of the site.

## What is the value of this?

Keep the work that’s been done on the same branch, so it’s easier to refer to, and less likely to disappear.

There were also CI + CD workflows, and a Dependabot check, which are left as-is.

## How to measure success?

We can still browser the history, and blame is intact: https://github.com/guardian/guardian-engineering-site/blame/mxdvl/keep-gatsby-around/gatsby/src/layouts/index.tsx

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+      matrix:
+        node-version: [16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn
+      - run: yarn ci
```
Comment (mxdvl) [inline .github/workflows/ci.yml:16]: Obviously we’ll need to update this to `pnpm` soon.

File: .github/workflows/cd.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - run: yarn
+      - run: yarn ci
+      - name: Build and deploy to gh-pages
+        uses: enriikke/gatsby-gh-pages-action@v2
```
Comment (mxdvl) [inline .github/workflows/cd.yml:21]: See #128 for the new Astro build steps.

**Review by mxdvl — COMMENTED** [review]
You just have to trust me that I’ve kept the files around.

**Review by frankie297 — APPROVED** [review]
👍

**Comment by mxdvl** [comment]
Thanks @adamnfish for the suggestion!

**Pull request merged**